apps/crn-server/test/fixtures/news.fixtures.ts

```diff
@@ -39,19 +80,26 @@ export const newsSquidexApiResponse: {
   ],
 };
 
-export const getNewsDataObject = (): NewsDataObject => ({
+export const getNewsDataObject = (
+  isContentfulResponse = false,
+): NewsDataObject => ({
   id: 'news-1',
   title: 'News 1',
   type: 'News',
+  frequency: 'News Articles',
   shortText: 'Short text of news 1',
   text: '<p>text</p>',
-  thumbnail: `${baseUrl}/api/assets/${appName}/thumbnail-uuid1`,
+  thumbnail: isContentfulResponse
```

Comment:
im not a huge fan of this - the DataObject should be the same for both. Now we know that contentful url's will be different but perhaps there is a way to store the base path in a config or something?

Reply:
I'm making the url the same as squidex then [a5631a3](https://github.com/yldio/asap-hub/pull/2247/commits/a5631a3cfdf93bd8a7858ba4bd8f6bc0863d0dac), is it ok?